tests/std/tests/P0896R4_ranges_subrange/test.compile.pass.cpp

```diff
@@ -1483,3 +1484,44 @@ namespace test_subrange {
     STATIC_ASSERT(test_tuple<subrange<int*, std::unreachable_sentinel_t, subrange_kind::sized>>());
     STATIC_ASSERT(test_tuple<subrange<int*, std::unreachable_sentinel_t, subrange_kind::unsized>>());
 } // namespace test_subrange
+
+namespace test_lwg_3589 {

```

Comment:
"LWG" is capitalized on 26.

Reply:
I can live with this, no need to reset testing.